<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MetersDateTimeButtonsWdgt</class>
 <widget class="QWidget" name="MetersDateTimeButtonsWdgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>60</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>All</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbCorrectionAll">
        <property name="text">
         <string>Time correction for all</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../Matilda-units/gui/image-and-font/res.qrc">
          <normaloff>:/katynko/svg3/chronometer-err.svg</normaloff>:/katynko/svg3/chronometer-err.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>21</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Selected</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="pbRead">
           <property name="text">
            <string>Read</string>
           </property>
           <property name="icon">
            <iconset resource="../../../../Matilda-units/gui/image-and-font/res.qrc">
             <normaloff>:/katynko/svg/chronometer.svg</normaloff>:/katynko/svg/chronometer.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbWrite">
           <property name="text">
            <string>Time correction</string>
           </property>
           <property name="icon">
            <iconset resource="../../../../Matilda-units/gui/image-and-font/res.qrc">
             <normaloff>:/katynko/svg3/chronometer-err.svg</normaloff>:/katynko/svg3/chronometer-err.svg</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>77</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../Matilda-units/gui/image-and-font/res.qrc"/>
 </resources>
 <connections/>
</ui>
